frontend/frontend-auto-translate.php

```diff
@@ -33,7 +33,7 @@ public function __construct( &$polylang ) {
 		$this->model = &$polylang->model;
 		$this->curlang = &$polylang->curlang;
 
-		add_action( 'pre_get_posts', array( $this, 'pre_get_posts' ) ); // after main Polylang filter
+		add_action( 'parse_query', array( $this, 'pre_get_posts' ), 100 ); // After all Polylang filters.
```

Comment:
I think it worths renaming `$this->pre_get_posts`, don't you think?
Also, do you see any edge case (like WC block that should be translated on the fly on front) to test?

Reply:
I had the same thought when I changed the filter, isn't it risky to rename a public method?

Reply:
We could still implement a depreciation, what do you think @Chouby ?